### PR TITLE
Python 3 compat.

### PR DESCRIPTION
**Pull request by thomasgoirand**
This patch adds Python3 compatibility to q.

### CODE REVIEW

File: bin/q [hunk 1]
```diff
@@ -1498,17 +1503,17 @@ class QOutputPrinter(object):
...
-                print >>f_out,"  `%s` - %s" % (n,t)
+                print("  `%s` - %s" % (n,t), file=f_out)
 
     def print_output(self,f_out,f_err,results):
         try:
             self._print_output(f_out,f_err,results)
-        except (UnicodeEncodeError, UnicodeError), e:
-            print >>f_err, "Cannot encode data. Error:%s" % e
+        except (UnicodeEncodeError, UnicodeError) as e:
+            print("Cannot encode data. Error:%s" % e, file=sys.stderr)
```
Comment (harelba) [inline bin/q]:
```suggestion
            print("Cannot encode data. Error:%s" % e, file=f_err)
```

File: bin/q [hunk 2]
```diff
@@ -1430,19 +1435,19 @@ def quote_minimal_func(output_delimiter,v):
     if v is None:
         return v
     t = type(v)
-    if t == str or t == unicode and ((output_delimiter in v) or ('"' in v)):
+    if t == str or t == six.text_type and ((output_delimiter in v) or ('"' in v)):
```
Comment (harelba) [inline bin/q]:
```suggestion
    if (t == str or t == six.text_type) and ((output_delimiter in v) or ('"' in v)):
```
This is a bug that i've found when experimenting with python 3 changes. If fixes the evaluation order to the expected one (unrelated specifically to p3)

**Comment by cclauss** [comment]
Can you try to resolve conflicts?

**Comment by harelba** [comment]
Closing this since #195 contains full python2 and python3 compatibility.

Thanks @thomasgoirand

**Pull request closed without merging**